src/coreclr/dlls/mscoree/mscorwks_unixexports.src

```diff
@@ -1,3 +1,6 @@
+; Android hosting API
+android_coreclr_initialize
```

Comment:
We should not be adding new public APIs to `coreclr.dll`.

Reply:
I agree, but I didn't want to make bigger changes and create a separate script for Android.